helix-core/src/main/java/org/apache/helix/manager/zk/ZKHelixManager.java

```diff
@@ -980,13 +997,13 @@ void waitUntilConnected() {
         continue;
       }
 
-      _sessionId = Long.toHexString(_zkclient.getSessionId());
+      _sessionId = _zkclient.getHexSessionId(_zkclient.getSessionId());
```

Comment:
Why do we need this?   if we really need a different method for Hex Id, then why not _zkClient.getHexSessionId directly return the Hex(_sessionId)? This embeded calls looks a bit weird